docs/ruleset.rst

```diff
@@ -45,11 +45,6 @@ Left-hand sides are currently a few special forms which select nodes to be fed t
 
 .. function :: when(predicate)
```

Comment:
The when() docs don't show up for me when I do the build (go into `docs`, `make html`, then look at `_build/html/index.html` in a browser). I notice there's a space before your ::. I don't know if ReST is that picky, but it wouldn't surprise me. Ah, and I see your when() comment starts with `/*`. It needs to start with `/**`. It's a jsdoc thing.